<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPClient</class>
 <widget class="QMainWindow" name="GPClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GP VPN Client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/logo.svg</normaloff>:/images/logo.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="statusImage">
        <property name="styleSheet">
         <string notr="true">#statusImage {
	image: url(:/images/not_connected.png);
	padding: 15
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Not Connected</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="portalInput">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Please enter your portal address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gatewaysComboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>260</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuCertificates">
     <property name="title">
      <string>Certificates</string>
     </property>
     <addaction name="actionInstall_Root_CA_s"/>
     <addaction name="actionUninstall_Root_CA_s"/>
    </widget>
    <addaction name="actionLogout"/>
    <addaction name="actionClear_data"/>
    <addaction name="separator"/>
    <addaction name="menuCertificates"/>
    <addaction name="actionUse_IPSec"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionQuit"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionClear_data">
   <property name="text">
    <string>Clear data</string>
   </property>
  </action>
  <action name="actionInstall_Root_CA_s">
   <property name="text">
    <string>Install Root CA's</string>
   </property>
  </action>
  <action name="actionUninstall_Root_CA_s">
   <property name="text">
    <string>Uninstall Root CA's</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionUse_IPSec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use IPSec</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>GPClient</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
